addons/getLoadoutsFromUnits/functions/fnc_getItems.sqf

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+params ["_items", "_tab"];
+
+private _return = [];
+private _count = count _items;
+
+if (_count > 0) then {
+    private _quadrupleTab = _tab + _tab + _tab + _tab;
+    {
+        private _amount = _x select ([1,2] select (count _x == 3));
```

Comment:
Just tested this. It looks to me like the amount is always `_x select 1` even for magazines. Here are some logs for `_x`:

```
19:07:20 ["fn_getItems",["FirstAidKit",1]]
19:07:20 ["fn_getItems",["30Rnd_65x39_caseless_mag",2,30]]
19:07:20 ["fn_getItems",["30Rnd_65x39_caseless_mag",5,30]]
19:07:20 ["fn_getItems",["16Rnd_9x21_Mag",2,17]]
19:07:20 ["fn_getItems",["HandGrenade",2,1]]
19:07:20 ["fn_getItems",["B_IR_Grenade",2,1]]
```

Reply:
okay, then they changed it. Makes it easier for us